platform/linux-generic/odp_schedule_iquery.c

```diff
@@ -1255,6 +1257,7 @@ static void order_unlock(void)
 static void schedule_order_lock(unsigned lock_index)
 {
 	odp_atomic_u64_t *ord_lock;
+	odp_atomic_u64_t *lock_acquired;
```

Comment:
I suspect same comments for this patch apply equally to all of the scheduler variants. Not repeating individual review comments here.